src/pages/FormPage.tsx

```diff
@@ -232,12 +260,21 @@ const FormPage = () => {
               </tds-radio-button>
             </section>
 
-            <section>
+            <section className="tds-u-mt3">
+              <h5>Textarea</h5>
               <tds-textarea
                 mode-variant="secondary"
-                name="textarea"
-                label="What do you do at Scania?"
+                name="occupationalEssay"
+                label="What do you do at Scania? (Minimum 50 chars)"
                 label-position="outside"
+                state={occupationEssayState}
+                helper={helperTextOccupationEssayState}
+                disabled={textareaDisabled === true}
```

Comment:
```suggestion
                disabled={textareaDisabled}
```
The === true isn't needed here